src/styles/home.scss

```diff
@@ -418,14 +421,8 @@
 // Team
 // ---------------------------------------------------------------------------
 .team__container {
-  height: 800px;
-
   @media (max-width: $xxs) {
-    padding: 0px 15px;
-  }
-
-  @media (min-width: $xl) and (max-width: 1399px) {
```

Comment:
Was having trouble finding the media query keyword that has the meaning of up to but not including (wanted to do something like max-width: 1400px but not include 1400px) 

Is there a keyword for that?

Reply:
hmm good question, not sure but you could probably do the inverse? ie move the conditional css outside the condition, and then move the stuff outside inside the condition, and then flip it

Reply:
cc @jzhzhang66